Scroller/useThemeScroller.js

```diff
@@ -202,8 +197,6 @@ const useSpottable = (props, instances) => {
 	 * Calculates the new top and left position for scroller based on focusedItem.
 	 *
 	 * @param {Node} item node
-	 * @param {Object} scrollInfo position info. `calculateScrollTop` uses
-	 * `scrollInfo.previousScrollHeight` and `scrollInfo.scrollTop`
```

Comment:
It seems to me that there is only one param which has `item` and `scrollPosition`. But you defined two params. One for the item node. One for the scroll position.

Reply:
I left the `scrollPosition` because it is the parameter for this function.
`function calculatePositionOnFocus ({item, scrollPosition}) {`

Reply:
OK.